go/obscuronode/node.go

```diff
@@ -185,22 +197,35 @@ func (a *Node) Stop() {
 }
 
 // Called only by the first enclave to bootstrap the network
-func (a *Node) initialiseProtocol() common.L2RootHash {
-	//todo shared secret
-	genesis := a.Enclave.ProduceGenesis()
-	tx := common.L1Tx{Id: uuid.New(), TxType: common.RollupTx, Rollup: common2.EncodeRollup(genesis.Rollup.ToRollup())}
+func (a *Node) initialiseProtocol() (common.L2RootHash, error) {
+	// todo shared secret
+	genesis, err := a.Enclave.ProduceGenesis()
```

Comment:
it doesn't make sense to handle failure at producing a genesis. 
It should have paniced earlier in the enclave

Reply:
Error propagation from the rlp hasher. Removing it 👍